src/harp_pedal_wizard.lua

```diff
@@ -0,0 +1,1591 @@
+function plugindef()
+    -- This function and the 'finaleplugin' namespace
+    -- are both reserved for the plug-in definition.
+    finaleplugin.RequireSelection = true
+    finaleplugin.Author = "Jacob Winkler"
+    finaleplugin.Copyright = "2022"
+    finaleplugin.Version = "2.0"
+    finaleplugin.Date = "2022-07-06"
+    return "Harp Pedal Wizard", "Harp Pedal Wizard", "Creates Harp Diagrams and Pedal Changes"
+end
+
+if finenv.IsRGPLua == false then
```

Comment:
This is a perfect example why you should never explicitly compare boolean values to `true` or (especially) `false`. This code does not do what you expect it to in JW Lua. That's because in JW Lua, `IsRGPLua` is `nil`, not `false`. So in JW Lua this expression will evaluate to false (that is `IsRGPLua` ~= `false`.)

Replace this with `if not finenv.IsRGPLua then`.

Reply:
Removed! I used the minimum version as you suggested elsewhere